### DISCUSSION TITLE
denied pull from ghcr.io while set to public

### DISCUSSION THREAD

**Discussion by hydazz** [community/community#43380] [Actions] [answered]
### Select Topic Area

Question

### Body

Im trying to pull https://github.com/imagegenius/docker-duplicati/pkgs/container/duplicati and im getting the below error,

```
root@Discovery:~# docker pull ghcr.io/imagegenius/duplicati
Using default tag: latest
Error response from daemon: Head "https://ghcr.io/v2/imagegenius/duplicati/manifests/latest": denied: denied
```

I have no idea what is causing this, the package is set to public. this applies to all packages under my organisation https://github.com/imagegenius/

**Comment by hydazz** [comment, marked as answer]
fixed by removing the docker credentials

**Reply by itsmethemojo** [reply]
access token used for login was expired. docker logout ghcr.io

thats 3 hours of my life i never get back

**Reply by marco-brandizi** [reply]
> thats 3 hours of my life i never get back

Thank you for sparing 3 hours of my life.

**Reply by neilenns** [reply]
Thank you for saving me hours of headache!!!

**Comment by IsoDevMate** [comment]
1. Logout from Docker registry - This is if you want to explicitly logout of  GitHub Container Registry (ghcr.io) 
   RUN :
      docker logout ghcr.io

3.You may manually remove the the docker credentials from your instance e.g.:
   RUN 
     cat ~/.docker/config.json //optional to check if the files is present or if it has any content 
      rm ~/.docker/config.json

Here is alink to the stack  various suggestions that  helped me out   https://stackoverflow.com/questions/70313502/error-response-from-daemon-get-https-ghcr-io-v2-denied-denied
